### DISCUSSION TITLE
None Conda installation

### DISCUSSION THREAD

**Discussion by Proteusiq** [pymc-devs/pymc3#6033] [General]
At the moment, there exists no stable way to install pymc>=4 without using `conda`.  Could we have instructions for at least one, none conda, way to install pymc for example: `poetry`  or pure .venv and pip?

**Comment by reshamas** [comment]
This might be helpful.

Reference:
https://twitter.com/edo_vivo/status/1534080216376696840?s=20&t=cHKqLXgMOjd4JzmiepHM4A

>Will this be available with pip?

>It already is, we just don't advertise it because it's more difficult to set up a compiler chain and MKL.

**Reply by Proteusiq** [reply]
@twiecki Is it possible to share the link to the not advertise none conda installation?

**Reply by twiecki** [reply]
Maybe a note could be helpful, yeah.

**Reply by Proteusiq** [reply]
That would be useful and grateful. We love conda, but in some cases, it feels overwhelming to install a whole tool(miniconda) just for PyMC
